tests/logging/enableApiLog.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/logging/enableApiLog.js
+ */
+'use strict'; // eslint-disable-line strict
+
+const supertest = require('supertest');
+const api = supertest(require('../../index').app);
+const tu = require('../testUtils');
+const u = require('./utils');
+const constants = require('../../api/v1/constants');
+const Aspect = tu.db.Aspect;
+const path = '/v1/samples/upsert/bulk';
+
+/* replicated api bulk upsert basic tests with enableApiLogs enabled
+ to execute api activity logging and test that it does not interfere with
+ the existing code. */
+describe('api: POST ' + path, () => {
```

Comment:
Good point, I will add more tests for other endpoints, as API logging will cover not only POST, but PUT, PATCH, DELETE.